server/util/db/db.go

```diff
@@ -578,10 +670,6 @@ func ParseDatasource(fileResolver fs.FS, datasource string, advancedConfig *Adva
 			return nil, err
 		}
 
-		if ac.Endpoint == "" {
```

Comment:
What's the reason for removing this check?

Reply:
I think it got moved and then removed from the place it was moved to; I will reinstate it.